modules/openxoutstreamBidAdapter.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+
+```
+Module Name: OpenX Outstream Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: opensource@yieldmo.com
+Note: Ads will only render in mobile
+```
+
+# Description
+
+Module that connects to OpenX's demand sources for outstream
```

Comment:
description could be more complete — also do we need to mention Yieldmo here?

Reply:
@harquail is this sufficient?

Reply:
👍